roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```diff
@@ -0,0 +1,180 @@
+---
+- name: Verify - Zookeeper User Change and custom logging path
+  hosts: zookeeper1-kafka-broker1
+  gather_facts: false
+  tasks:
+    - name: Set Zookeeper user to new user
+      set_fact:
+        zookeeper_user: zk-test-user
+
+    - name: Set Zookeeper Group
+      set_fact:
+        zookeeper_group: confluent-custom
+
+    - name: Create Zookeeper User
+      user:
+        name: "{{zookeeper_user}}"
+        comment: "Zookeeper User"
+        system: true
+        group: "{{zookeeper_group}}"
+
+    - name: Create Log Directory
+      file:
+        path: "{{zookeeper.log_path}}"
+        state: directory
+        group: "{{zookeeper_group}}"
+        owner: "{{zookeeper_user}}"
+        mode: 0770
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{zookeeper_service_name}}"
+        state: restarted
+
+    - import_role:
+        name: confluent.zookeeper
+        tasks_from: health_check.yml
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{kafka_broker_service_name}}"
+        state: restarted
+
+    - name: Wait for Under Replicated Partitions on Broker
+      include_tasks: ../../../../tasks/wait_for_urp.yml
+    
+    - import_role:
+        name: confluent.test
+        tasks_from: check_directory.yml
+      vars:
+        custom_path: /kafka/logs/zookeeper/
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_directory.yml
+      vars:
+          custom_path: /kafka/logs/kafka/
+          
+- name: Verify - Schema Registry custom user and custom group and custom logging directory
+  hosts: schema_registry
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: sr-cust+
+        expected_value: sr-cust+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: sr-custom
+        group: sr-custom-group
+    
+    - import_role:
```

Comment:
We have pretty good confidence that the directory will get created because that is an ansible task, but how to we verify that log files are written within that directory?